tests/Type/InputObjectTypeTest.php

```diff
@@ -0,0 +1,132 @@
+<?php declare(strict_types=1);
+
+namespace GraphQL\Tests\Type;
+
+use GraphQL\Examples\Blog\Types;
+use GraphQL\GraphQL;
+use GraphQL\Type\Definition\InputObjectType;
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Definition\Type;
+use GraphQL\Type\Schema;
+use PHPUnit\Framework\TestCase;
+
+class Tag
+{
+    public string $name;
+
+    public string $value;
+
+    public function __construct(string $name, string $value)
+    {
+        $this->name = $name;
+        $this->value = $value;
+    }
+}
+
+class StoryFiltersInput
+{
+    public string $author;
+
+    public bool $popular;
+
+    public Tag $tag;
+
+    public function __construct(string $author, bool $popular, Tag $tag)
+    {
+        $this->author = $author;
+        $this->popular = $popular;
+        $this->tag = $tag;
+    }
+}
+
+class InputObjectTypeTest extends TestCase
+{
+    public function testParseValue(): void
```

Comment:
Let's add a test that ensures `parseValue` is not called when explicitly passing `null` as the value for an input object.